packages/changed-elements-react/src/widgets/comparisonJobWidget/models/ComparisonJobModels.ts

```diff
@@ -31,3 +33,14 @@ export type JobStatusAndJobProgress = {
   jobStatus: JobStatus;
   jobProgress: JobProgress;
 };
+
+export type JobId = string;
```

Comment:
Unless change in `JobId` type is already planned, please just use a plain string instead. Type aliases in TypeScript add no safety and increase mental load on person reading the code.